Tests/PayID/Fakes/FakePayIDClient.swift

```diff
@@ -2,9 +2,10 @@ import Foundation
 import XpringKit
 
 /// Fakes a PayID client.
-public class FakePayIDClient: PayIDClientProtocol {
+public class FakePayIDClient: XRPPayIDClientProtocol {
   /// Results from method calls.
   private let addressResult: Result<String, PayIDError>
+  public let xrplNetwork: XRPLNetwork = .test
```

Comment:
Optional: 
This obviously works, but I'd choose to inject it in the initializer in case anyone wanted to customize it later.

Reply:
looks like this happened upstream